benchmark/CMakeLists.txt

```diff
@@ -15,6 +15,15 @@ target_link_libraries(histogram_benchmark
                       scipp-core
                       scipp_test_helpers)
 
+add_executable(neutron_convert_benchmark EXCLUDE_FROM_ALL neutron_convert_benchmark.cpp)
+add_dependencies(all-benchmarks neutron_convert_benchmark)
```

Comment:
Is there a good reason for not having a single executable for the benchmarks?

Reply:
I don't know, might be worth looking into.